src/common/config.hpp

```diff
@@ -22,10 +22,15 @@
 struct ActuatorMessage {
   bool ov101; // Ox Fill
   bool ov102; // Ox Drain
-  bool nv101; // N2 Purge
-  bool ov201; // Vent Isolation
-  bool ov202; // Vent Solenoid
-  bool injector_valve;
+  bool nv201; // N2 Purge
+  bool nv202; // N2 Drain (Solenoid)
```

Comment:
I believe this is a motor valve and it's also QD disconnect, not just drain. Double check with Matthew tho.